script.js

```diff
@@ -0,0 +1,295 @@
+  ],
+  [
+    ['ControlLeft', 'Ctrl', 'Ctrl', 'Ctrl', 'Ctrl'],
+    ['MetaLeft', 'Win', 'Win', 'Win', 'Win'],
+    ['AltLeft', 'Alt', 'Alt', 'Alt', 'Alt'],
+    ['Space', ' ', ' ', ' ', ' '],
+    ['AltRight', 'Alt', 'Alt', 'Alt', 'Alt'],
+    ['ArrowLeft', '◄', '◄', '◄', '◄'],
+    ['ArrowDown', '▼', '▼', '▼', '▼'],
+    ['ArrowRight', '►', '►', '►', '►'],
+    ['ControlRight', 'Ctrl', 'Ctrl', 'Ctrl', 'Ctrl'],
+  ],
+];
+
+const wrapper = document.createElement('div'); // задаёт элемент с заданным тегом
+const textArea = document.createElement('textarea');
+const keyBoard = document.createElement('div');
+let lang = 'rus';
+let capslock = false;
+wrapper.className = 'wrapper';// задаёт имя класса
+textArea.id = 'textArea';
+keyBoard.className = 'keyBoard';
+wrapper.append(textArea);
+wrapper.append(keyBoard);
+for (let i = 0; i < Layout.length; i++) {
+  const row = document.createElement('div');
+  row.classList.add('row');
+  for (let j = 0; j < Layout[i].length; j += 1) {
+    const key = document.createElement('div');
+    key.classList.add('key');
+    key.classList.add(Layout[i][j][0]);
+
+    key.insertAdjacentHTML('afterBegin',
+      `<div class='rus'>
+          <span class='caseDown '>${Layout[i][j][1]}</span>
+          <span class='caseUp hidden'>${Layout[i][j][2]}</span>
+        </div>
+        <div class='eng hidden'>
+          <span class='caseDown hidden'>${Layout[i][j][3]}</span>
+          <span class='caseUp hidden'>${Layout[i][j][4]}</span>
+        </div>`);
+    row.appendChild(key);
+  }
+  keyBoard.appendChild(row);
+}
+
+function addActive(elem) {
+  elem.classList.add('active');
+}
+
+function asincron(func, milliseconds) {
+  let lastCall = 0;
+  return function () {
+    const now = Date.now();
+    if (lastCall + milliseconds < now) {
+      lastCall = now;
+      return func.apply(this, arguments);
+    }
+  };
+}
+
+function removeActive(elem) {
+  elem.classList.remove('active');
+}
+const changeCase = () => {
+  const langElem = keyBoard.querySelectorAll(`div > .${lang}`);
+  console.log(langElem);
+  for (let i = 0; i < langElem.length; i++) {
+    langElem[i].querySelectorAll('span')[0].classList.toggle('hidden');
+    langElem[i].querySelectorAll('span')[1].classList.toggle('hidden');
+  }
+};
+
+const changeLang = () => {
+  const prevLang = keyBoard.querySelectorAll(`div > .${lang}`);
+  for (let i = 0; i < prevLang.length; i++) {
+    prevLang[i].classList.toggle('hidden');
+    prevLang[i].querySelectorAll('span')[0].classList.toggle('hidden');
+  }
+  if (lang === 'rus') {
+    lang = 'eng';
+    localStorage.setItem('lang', lang);
+  } else {
+    lang = 'rus';
+    localStorage.setItem('lang', lang);
+  }
+  const nextLang = keyBoard.querySelectorAll(`div > .${lang}`);
+  for (let i = 0; i < nextLang.length; i += 1) {
+    nextLang[i].classList.toggle('hidden');
+    nextLang[i].querySelectorAll('span')[0].classList.toggle('hidden');
+  }
+};
+
+if (localStorage.lang === 'eng') {
+  changeLang();
```

Comment:
You could use one source of trues, for example `localStorage`:
```javascript
// start app
const defaultLang = 'rus';
const currentLang = localStorage.lang || defaultLang;

changeLang(currentLang);
...
```

Inside `changeLang` you can save new lang to `localStorage`